packages/core/src/modules/employment/use-cases/send-company-review-notifications.ts

```diff
@@ -0,0 +1,63 @@
+import dedent from 'dedent';
+
+import { db } from '@oyster/db';
+
+import { job } from '@/infrastructure/bull/use-cases/job';
+import { ENV } from '@/shared/env';
+
+/**
+ * This is a weekly job that runs and finds all students that have had a work
+ * experience end in the last week. Then, it has the ColorStack bot send a DM
+ * to all these students suggesting that they add a review of their experience.
+ */
+export async function sendCompanyReviewNotifications() {
+  const oneMonthAgo = new Date();
+
+  oneMonthAgo.setDate(oneMonthAgo.getDate() - 32);
+
+  const results = await db
+    .selectFrom('workExperiences')
+    .leftJoin('students', 'students.id', 'workExperiences.studentId')
+    .select([
+      'workExperiences.companyId',
+      'workExperiences.companyName',
+      'students.firstName as studentName',
+      'students.slackId as studentSlackId',
+    ])
+    .where('endDate', '>=', oneMonthAgo)
+    .where('endDate', 'is not', null)
```

Comment:
Since `endDate` could be some time in the future, we also need to have a check to ensure `endDate` has already occurred.